src/components/Navbar/Navbar.scss

```diff
@@ -1,4 +1,8 @@
-@import "../../css/common.scss";
+@import "../../css/variables.scss";
+
+.navbar-ams{
+  padding-top: 2.2rem!important;
```

Comment:
I think we probably don't need the !important here now that we found the css duplication.